<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_info</class>
 <widget class="QMainWindow" name="modify_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>-260</y>
      <width>1381</width>
      <height>1341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>First name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Last name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>phonenumber: +20</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fname_new">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lname_new">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_new">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_new">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Rewards:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rewards_output">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_fname">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>81</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_lname">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_phone">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_pass">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>modify</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Wallet balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="walletbalance_output">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>640</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../intro to eng/31WJggtqtXL.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="setcreditcard">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fname_label">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>53</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lname_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="phone_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pass_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1167</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuprofile">
    <property name="title">
     <string>profile</string>
    </property>
   </widget>
   <addaction name="menuprofile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
